README.md

```diff
@@ -107,16 +103,24 @@ GLOBAL OPTIONS:
    * This contains binaries for supported architectures.
 3. Review Help Documentation:
    * Execute: `./operator-<RELEASE_VERSION>-<ARCH> --help`, replacing ARCH with your actual architecture and <RELEASE_VERSION> with latest release.
-4. Run the Operator:
+4. Register with Aethos AVS:
+   ```sh 
+   export AETHOS_SIGNING_ADDRESS="aethos_avs_signing_address"
+   export REGISTRATION_PRIVATE_KEY="eigenlayer_registered_operator_private_key"
+   export ETH_RPC_URL="eth_rpc_url"
+
+   docker run ghcr.io/aethosnetwork/operator:latest register --config=/app/config.yaml --aethos-signing-address=${AETHOS_SIGNING_ADDRESS} --registration-private-key=${REGISTRATION_PRIVATE_KEY} --eth-rpc-url=${ETH_RPC_URL}
```

Comment:
update this to `./operator-<RELEASE_VERSION>-<ARCH>` instead of `docker run <image>`